docs/docs/guide/02-hello.md

```diff
@@ -3,42 +3,51 @@ sidebar_position: 2
 description: Step-by-step guidance to build your first blockchain and your first Cosmos SDK module. 
 ---
 
-# Hello, Ignite CLI 
+# Hello, Ignite CLI
 
-This tutorial is a great place to start your journey into the Cosmos ecosystem. Instead of wondering how to build a blockchain, follow these steps to build your first blockchain and your first Cosmos SDK module.
+This tutorial is a great place to start your journey into the Cosmos ecosystem. Instead of wondering how to build a
+blockchain, follow these steps to build your first blockchain and your first Cosmos SDK module.
 
 ## Get started
 
-In the previous chapter you've learned how to install [Ignite CLI](https://github.com/ignite/cli), the tool that offers everything you need to build, test, and launch your blockchain with a decentralized worldwide community.
+In the previous chapter you've learned how to install [Ignite CLI](https://github.com/ignite/cli), the tool that offers
+everything you need to build, test, and launch your blockchain with a decentralized worldwide community.
 
-This series of tutorials is based on a specific version of Ignite CLI, so be sure to install the correct version. For example, to install Ignite CLI v0.22.2 use the following command:
+This series of tutorials is based on a specific version of Ignite CLI, so be sure to install the correct version. For
+example, to install Ignite CLI v0.22.2 use the following command:
```

Comment:
My intention with #2902 is to remove any version constraints, so the user always use the latest version to build all the tutorials. I understand that this adds additional work when we publish a release (checking the tutorials), there's an issue for investigating on how we could test the tutorial automatically (#2593).

Anyway if you prefer to restrict the version, then you should change v0.22.2 to v0.25 in this sentence.

Reply:
see comment above :)